### PR TITLE
Update Rust Courses

### PR DESCRIPTION
**Pull request by alok-cmd**
Added Rust Courses which were good in quality on YouTube and beginner friendly.

## What does this PR do?
Add Resource

### Why is this valuable (or not)?
Most People can't find good courses of Rust on YouTube as its hard to explain.

### How do we know it's really free?
I've tested every resource I added.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by alok-cmd** [comment]
I forgot to select them. Sorry

**Comment by JojiiOfficial** [comment]
This is probably related to #5039

**Comment by eshellman** [comment]
Be sure to check the output of Travis-CI for linter errors. Alphabetical order and spacing are the most common problems.

**Comment by alok-cmd** [comment]
> Be sure to check the output of Travis-CI for linter errors. Alphabetical order and spacing are the most common problems.

Okay

File: free-courses-en.md [hunk 1]
```diff
@@ -583,6 +584,14 @@
...
 * [RubyMonk - Interactive Ruby tutorials](https://rubymonk.com)
 
+### Rust
+
+* [Intro to Rust](https://www.youtube.com/playlist?list=PLJbE2Yu2zumDF6BX6_RdPisRVHgzV02NW) - Tensor Programming (YouTube)
+* [Rust Basics](https://www.youtube.com/playlist?list=PLlcnQQJK8SUjApd95LIcd3K9XXmE-IeCS) - Engineer Man (YouTube)
+* [Rust Crash Course](https://www.youtube.com/watch?v=zF34dRivLOw) - Traversy Media (YouTube 1.5hr)
+* [Rust Projects](https://www.youtube.com/playlist?list=PLJbE2Yu2zumDD5vy2BuSHvFZU0a6RDmgb) - Tensor Programming (YouTube)
+* [Rust Tutorial](https://www.youtube.com/playlist?list=PLLqEtX6ql2EyPAZ1M2_C0GgVd4A-_L4_5) - Doug Milford (YouTube)
+
```
Comment (kadhirash) [inline free-courses-en.md]: Hello! Thank you for the resources

Currently, the build is failing due to not having an extra blank line after the last resource in the Ruby section and before your Rust heading.

Check the reason why [here](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)

Also, no need to mention (Youtube), check that [here](https://github.com/EbookFoundation/free-programming-books/issues/4579#issuecomment-716214451)


```suggestion
* [Ruby Tutorial](https://www.w3resource.com/ruby/)
* [RubyMonk - Interactive Ruby tutorials](https://rubymonk.com)


### Rust

* [Intro to Rust](https://www.youtube.com/playlist?list=PLJbE2Yu2zumDF6BX6_RdPisRVHgzV02NW) - Tensor Programming
* [Rust Basics](https://www.youtube.com/playlist?list=PLlcnQQJK8SUjApd95LIcd3K9XXmE-IeCS) - Engineer Man
* [Rust Crash Course](https://www.youtube.com/watch?v=zF34dRivLOw) - Traversy Media
* [Rust Projects](https://www.youtube.com/playlist?list=PLJbE2Yu2zumDD5vy2BuSHvFZU0a6RDmgb) - Tensor Programming
* [Rust Tutorial](https://www.youtube.com/playlist?list=PLLqEtX6ql2EyPAZ1M2_C0GgVd4A-_L4_5) - Doug Milford

```

**Review by kadhirash — CHANGES_REQUESTED** [review]

File: free-courses-en.md (hunk 1 above)
Comment (kadhirash) [reply free-courses-en.md]: You also didn't need to remove that blank line in your recent commit, that was fine before!

Comment (alok-cmd) [reply free-courses-en.md]: Thanks kadhirash. I was getting confused about that blank lines as I didn't pay attention.

Comment (kadhirash) [reply free-courses-en.md]: Np, anytime!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**